elftools/dwarf/locationlists.py

```diff
@@ -106,7 +106,7 @@ def parse_from_attribute(self, attr, dwarf_version):
 
     @staticmethod
     def _attribute_has_loc_expr(attr, dwarf_version):
-        return ((dwarf_version < 4 and attr.form == 'DW_FORM_block1' and
+        return ((dwarf_version < 4 and attr.form.startswith('DW_FORM_block') and
```

Comment:
Is this correct?

The existing code seems to be specifically targeting `DW_FORM_block1`, not the other kinds; why is this changing? What new forms are needed for DWARFv1?

Reply:
Saw a location expression of form`DW_FORM_block2` in the reference binary. There was no `block1` in DWARF1, and no locations sections either.

If this is a dealbreaker, we can leave this one out and I'll monkeypatch.